client/my-sites/stats/activity-log/rewind-alerts.jsx

```diff
@@ -0,0 +1,39 @@
+/** @format */
```

Comment:
Why do we need to wrap the `ThreatAlert`s with this? It feels like it would be much simpler to just output them directly.

Reply:
If you are asking about the `RewindAlerts` component it's because I felt like the individual alerts were going to be (and already are) complicated enough to split up the files. I started just outputting them here but it grew beyond what my mind could comprehend and then I didn't see too much value in having them merged.

is this what you mean? do you think it would be better merged into a single file? (we know we have a diff view of files and also a highlighted view of files coming too into the individual alerts per threat-file)